server/routerlicious/packages/protocol-definitions/src/config.ts

```diff
@@ -18,12 +18,41 @@ export interface ISummaryConfiguration {
     maxAckWaitTime: number;
 }
 
+// Deli lambda configuration
+export interface IDeliServerConfiguration {
```

Comment:
If these doesn't go over the wire and server specific, they shouldn't be in the protocol-definitions.   
Should be good to move it some where else if a good place exist, or create on if it doesn't.  You can do that in the next PR